lib/srv/db/elasticsearch/engine.go

```diff
@@ -171,10 +186,170 @@ func (e *Engine) process(ctx context.Context, sessionCtx *common.Session, req *h
 		return trace.Wrap(err)
 	}
 	defer resp.Body.Close()
+	defer req.Body.Close()
```

Comment:
If `req.Body` needs to be closed this function can exist earlier without calling  `req.Body.Close()` like in case of 
copyRequest error